macros/shared

```diff
@@ -196,6 +196,7 @@ CROSS_COMPILATION_CONF_EOF\
   GOPROXY="off"; export GOPROXY ; \
   GOSUMDB="off"; export GOSUMDB ; \
   GO111MODULE="auto"; export GO111MODULE ; \
+  GO_VERSION="1.21.9"; export GO_VERSION ; \
```

Comment:
Do we still need this?

Reply:
no... thanks, accidentally let this sneak in..